src/app/component/ProductReviewList/ProductReviewList.component.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { ProductType } from 'Type/ProductList';
+import ContentWrapper from 'Component/ContentWrapper';
+import TextPlaceholder from 'Component/TextPlaceholder';
+import ProductReviewRating from 'Component/ProductReviewRating';
+import './ProductReviewList.style';
+
+/**
+ * @class ProductReviewList
+ */
+class ProductReviewList extends Component {
+    renderReviewListItemRating(ratingVoteItem) {
+        const {
+            vote_id, rating_code, percent
+        } = ratingVoteItem;
+
+        return (
+            <div
+              key={ vote_id }
+              block="ProductReviewList"
+              elem="RatingSummaryItem"
+            >
+                <span><TextPlaceholder content={ rating_code } /></span>
```

Comment:
<TextPlaceholder> is span itself. Please use content-fully meaningful element for rating code display.